### ISSUE TITLE
Contrast Issue in collection cards

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#472]
## Problem

![84492382-bb591880-acc3-11ea-9668-2dc7825ee68c](https://user-images.githubusercontent.com/45792768/84810983-c5637a00-b029-11ea-9e99-2d9152829b7d.png)

## Description

Currently, this span has a contrast ratio of **2.05** whereas the desired minimum contrast ratio should be **4.5** or **AA/AAA large**

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by ayanchoudhary** [comment]
@panchovm @dhruvkb

**Comment by zackkrida** [comment]
I think we could bump up to the next darkest gray in the design language and still maintain the spirit of the design.

**Comment by fcoveram** [comment]
For now, and after redesigning this page, we are not planning to use this component. I will not work on this until we need it again. What do you think?

**Comment by zackkrida** [comment]
I will close this! We'lll see if the cards ever make a re-appearance and we will adjust them then.

**Comment by ayanchoudhary** [comment]
Yes we can drop for now, since the page is going to be redesigned, so we will not need this component anymore
